llvm/lib/Transforms/Utils/BuildLibCalls.cpp

```diff
@@ -1206,37 +1209,18 @@ bool llvm::inferNonMandatoryLibFuncAttrs(Function &F,
   case LibFunc_expm1:
   case LibFunc_expm1f:
   case LibFunc_expm1l:
-  case LibFunc_fabs:
-  case LibFunc_fabsf:
-  case LibFunc_fabsl:
   case LibFunc_fdim:
   case LibFunc_fdiml:
   case LibFunc_fdimf:
-  case LibFunc_ffs:
-  case LibFunc_ffsl:
-  case LibFunc_ffsll:
   case LibFunc_floor:
   case LibFunc_floorf:
   case LibFunc_floorl:
```

Comment:
It looks like you dropped some of the calls that don't set errno, but not all of them? E.g. floor and nearbyint are still here.

Reply:
Ugh, was looking at a [older](https://pubs.opengroup.org/onlinepubs/007904975/functions/floor.html) version of POSIX. Indeed those do not set errno anymore in recent versions. Updated, thanks!